tools/sdk-testgen/packages/autorest.testmodeler/src/core/model.ts

```diff
@@ -239,6 +239,17 @@ export class ExampleModel {
     }
 }
 
+export class OutputVariableModel {
+    languageName: string;
+    public constructor(public kind: OutputVariableModelKind, public value: number | string | Languages) {}
```

Comment:
Union type is not convenient in strong typing languages (like .Net), so should we as far as possible avoid to use it in testmodeler?
For this feature, structure like below will be more friendly for .Net like consumer:
~~~
export class OutputVariableModel {
  indexValue?: number;
  keyValue?: string;
  objectValue?: Language;
}
~~~